fonts/roboto.css

```diff
@@ -0,0 +1,6 @@
+@font-face {
+    font-family: 'roboto';
+    src: url('roboto-medium-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/HTML_SNN/pullRequest?prid=6244016)